src/main/java/Duke.java

```diff
@@ -1,10 +1,423 @@
+        int taskIndex = Integer.parseInt(userInput.substring(userInput.indexOf(" ") + 1));
+        markTaskAsDone(taskIndex);
+    }
+
+    /**
+     * Prints new lines to make it look like the command window was cleared
+     */
+    private static void executeClearWindow () {
+        int newLinesToPrint = 100;
+
+        for (int i = 0; i < newLinesToPrint; i++) {
+            System.out.println();
+        }
+    }
+
+    /**
+     * Prints the descriptions on how to use each one of the supported commands
+     */
+    private static void executeHelp() {
+        String msg = "Below are descriptions of the supported commands:" + System.lineSeparator();
+
+        // help for exit command
+        msg += System.lineSeparator() + String.format("\t%s: exits the program. " +
+                "Input must follow the format below,", CMD_EXIT);
+        msg += System.lineSeparator() + "\t\tbye";
+        msg += System.lineSeparator();
+
+        // help for done command
+        msg += System.lineSeparator() + String.format("\t%s: marks task as done. " +
+                "Input must follow the format below,", CMD_DONE);
+        msg += System.lineSeparator() + "\t\tdone [task number]";
+        msg += System.lineSeparator();
+
+        // help for done command
+        msg += System.lineSeparator() + String.format("\t%s: lists the tasks currently available. " +
+                "Input must follow the format below,", CMD_LIST);
+        msg += System.lineSeparator() + "\t\tlist";
+        msg += System.lineSeparator();
+
+        // help for deadline command
+        msg += System.lineSeparator() + String.format("\t%s: adds deadline to your list of tasks. " +
+                "Input must follow the format below,", CMD_ADD_DEADLINE);
+        msg += System.lineSeparator() + "\t\tdeadline [description] /by [date/time]";
+        msg += System.lineSeparator();
+
+        // help for event command
+        msg += System.lineSeparator() + String.format("\t%s: adds event to your list of tasks. Input " +
+                "must follow the format below,", CMD_ADD_EVENT);
+        msg += System.lineSeparator() + "\t\tevent [description] /at [date/time]";
+        msg += System.lineSeparator();
+
+        // help for to do command
+        msg += System.lineSeparator() + String.format("\t%s: add todo to your list of tasks. Input" +
+                " must follow the format below,", CMD_ADD_TODO);
+        msg += System.lineSeparator() + "\t\ttodo [description]";
+        msg += System.lineSeparator();
+
+        // help for clear window
+        msg += System.lineSeparator() + String.format("\t%s: clears command window. Input" +
+                " must follow the format below,", CMD_CLEAR_WINDOW);
+        msg += System.lineSeparator() + "\t\tclear";
+        msg += System.lineSeparator();
+
+        replayBack(msg);
+    }
+
+
+    // ASCII art from: https://www.asciiart.eu/computers/bug, by Joan Start
+    private static void executeSpotBug () {
```

Comment:
I think it is interesting that you decided to create this function in order to help you find bugs. What kind of bugs have you been able to catch using it?

Reply:
Oh the function doesn't really find any bugs in the code 😅 it is an "easter egg" kind of thing that just prints an ASCII bug if the user inputs "debug" in the program. I'll change the name of the command and add comments. Or maybe I'll add it later so that when the program crashes it just prints it.